packaging/fedora/snapd.spec

```diff
@@ -426,6 +442,12 @@ rm -rf vendor/*
 # Extract each tarball properly
 %setup -q -D -b 1
 %endif
+%if %{with apparmor}
+APPARMOR_CFLAGS=-I/usr/include
+APPARMOR_LIBS=-lapparmor
+export APPARMOR_CFLAGS
+export APPARMOR_LIBS
```

Comment:
This is absolutely horrible, and also wouldn't actually persist, since it's in the wrong script stage.

Are you sure this is how this is supposed to work? I don't think it is...

Reply:
This was old and I meant to remove it.